dsa/Advance/Branch and Bound.md

```diff
@@ -0,0 +1,115 @@
+---
+id: 06 - Branch and Bound
```

Comment:
replace `id: 06 - Branch and Bound` to `id: branch-and-bound`

Reply:
@Ajay-Dhangar Made it